atmcfg/atmcfg_test.go

```diff
@@ -62,6 +64,156 @@ func TestStartup(t *testing.T) {
 	})
 }
 
+func TestShutdownProcessesByClusterName(t *testing.T) {
+	t.Run("replica set", func(t *testing.T) {
+		config := automationConfigWithOneReplicaSet(clusterName, false)
+		err := ShutdownProcessesByClusterName(config, clusterName, []string{"host0:27017"})
+		if err != nil {
+			t.Error(err)
```

Comment:
[nit] two things, first, you may want Fatal instead of Error, Error will go forward to the next check and I don't belive there's a point to check if the value changed if there was an error, second, just doing `t.Error(err)` is not going to be helpful if this test starts failing, it'll print the error and just that no context on what the problem with the test is, consider adding some context
```suggestion
			t.Fatalf("ShutdownProcessesByClusterName() returned an unexpected error: %v", err)
```